src/sys/socket/addr.rs

```diff
@@ -702,7 +702,7 @@ pub mod sys_control {
             let addr = sockaddr_ctl {
                 sc_len: mem::size_of::<sockaddr_ctl>() as c_uchar,
                 sc_family: AddressFamily::System as c_uchar,
-                ss_sysaddr: consts::AF_SYS_CONTROL as uint16_t,
```

Comment:
`sockaddr_ctl` is correctly defined. It's defined in macos [here](https://github.com/opensource-apple/xnu/blob/53c5e2e62fc4182595609153d4b99648da577c39/bsd/sys/kern_control.h#L137-L144), `ss_sysaddr` is indeed `u_int16_t`.

I do not know if `AF_SYS_CONTROL` is used anywhere else.
We could still make that change in libc as it has not been released yet.